.golangci.yml

```diff
@@ -0,0 +1,200 @@
+run:
```

Comment:
Would you please add the reference? It looks like this file cited externally.

Reply:
`.golangci.yml `  required by https://github.com/golangci/golangci-lint